src/lib/y2firewall/clients/firewall_new.rb

```diff
@@ -33,12 +33,18 @@ class FirewallNew
       include Yast::I18n
       include Yast::Logger
 
+      # Constructor
       def initialize
         textdomain "firewall"
+
+        Yast.import "UI"
```

Comment:
Unless there is a strong reason against that, you should move `import` calls next to regular Ruby `require` calls. Actually, they are mostly *requires*.